### PR TITLE
Metricbeat 5.0.0 (new formula)

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: Formula/metricbeat.rb [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  desc "Collect metrics from your systems and services."
+  homepage "https://www.elastic.co/products/beats/metricbeat"
+  url "https://github.com/elastic/beats/archive/v5.0.0.tar.gz"
+  sha256 "3e6b7cf2ee5f52e78ae87ef04ab9dd49977c89f86a09416586896aeaea844e34"
+
+  head "https://github.com/elastic/beats.git"
+
+  bottle do
+    cellar :any_skip_relocation
+  end
```
Comment (bfontaine) [inline Formula/metricbeat.rb]: Please leave out the `bottle do` block; our bot will handle it.

File: Formula/metricbeat.rb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        metricsets: ["load"]
+        period: 1s
+      output.file:
+        enabled: true
+        path: #{testpath}/data
+        filename: metricbeat
+    EOS
+
+    mkdir testpath/"data"
+    mkdir testpath/"logs"
```
Comment (bfontaine) [inline Formula/metricbeat.rb]: Use `(testpath/"logs").mkpath` instead.

File: Formula/metricbeat.rb [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+    metricbeat_pid = fork { exec bin/"metricbeat", "-c", testpath/"metricbeat.yml", "--path.data=#{testpath}/data", "--path.logs=#{testpath}/logs" }
```
Comment (bfontaine) [inline Formula/metricbeat.rb]: Please break this on multiple lines.

File: Formula/metricbeat.rb (hunk 2 above)
Comment (bfontaine) [inline Formula/metricbeat.rb]: Also why is there both a `log` file and a `logs` directory here?

**Review by bfontaine — CHANGES_REQUESTED** [review]

File: Formula/metricbeat.rb (hunk 1 above)
Comment (ruflin) [inline Formula/metricbeat.rb]: removed

File: Formula/metricbeat.rb (hunk 2 above)
Comment (ruflin) [inline Formula/metricbeat.rb]: Implementation changed to `mkpath`

I removed the log file as it is not needed. I forgot to remove it as I copied the initial code from the filebeat Formula.

**Comment by ruflin** [comment]
@bfontaine Thanks for the review. I pushed a new commit with the requested changes. Let me know if you prefer that I squash it into 1 commit before merging.

**Comment by bfontaine** [comment]
> Let me know if you prefer that I squash it into 1 commit before merging.

That’d be great, thanks!

**Comment by ruflin** [comment]
@bfontaine rebased and squashed

**Review by bfontaine — APPROVED** [review]

**Comment by dunn** [comment]
Thanks for your patience! Merged as 4e805850ee83bff5f82af6413e9d52695ab8dada.

**Comment by ruflin** [comment]
@dunn Thanks a lot for getting metricbeat into homebrew.

**Pull request closed without merging**